### ISSUE TITLE
Regression: HTTP auth prompt is not resized when there are multiple matching entries

### ISSUE THREAD

**Issue by dessant** [luckyrat/KeeFox#732]
With KeeFox 1.6.4 on Firefox 50, the HTTP auth prompt is not resized when KeeFox inserts the entry selector dropdown.

![untitled-1](https://cloud.githubusercontent.com/assets/7513068/20476135/0660c28e-afd8-11e6-89b6-58c64cbefb04.png)

**Comment by dlech** [comment]
It looks like this is a [Mozilla bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1277895). It is marked as fixed. However, they really didn't fix the root problem, which is mentioned in [comment 8](https://bugzilla.mozilla.org/show_bug.cgi?id=1277895#c8) - that `window.sizeToContent()` does not actually do what it says.

Maybe [one of these bugs about sizeToContent](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=sizeToContent+Status%3A) will provide some insight? If not, I would suggest filing a new bug with Mozilla.

**Comment by luckyrat** [comment]
Yeah looks like Mozilla just worked around the bug for their use case but that's no good for add-ons. It looks like this workaround was the cause of the last minute change in FF50 beta which caused #709 too.

**Comment by dlech** [comment]
I tested this with Firefox developer edition 52.0a2 on Linux and it seems to be fixed. So, this is probably going to be a "won't fix" in KeeFox and we will just have to deal with it for one or two firefox versions (unless mozilla fixes it in one of the 50.0.x updates).

**Comment by dessant** [comment]
Made a new test today on Windows 10 with Firefox 51.0.1, 52.0b1, 52.0a2 and 54.0a1. They all show the same issue, the popup is not resized.

**Comment by BoffinbraiN** [comment]
This issue is still present in Firefox 55.0 and KeeFox 1.7. Screenshots attached.

![keepass auth 1](https://user-images.githubusercontent.com/13255058/29064690-ac4f1cc0-7c21-11e7-9b77-dc7e5f3bd3a1.png)

![keepass auth 2](https://user-images.githubusercontent.com/13255058/29064693-aef792ae-7c21-11e7-85fb-fab5b8dedfce.png)
